account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -14,10 +14,15 @@ import {
 import { local070Instance } from "~test/instances.js";
 import { setBalance } from "viem/actions";
 import { accounts } from "~test/constants.js";
-import { getDefaultSingleSignerValidationModuleAddress } from "../modules/utils.js";
+import {
+  getDefaultSingleSignerValidationModuleAddress,
+  getDefaultAllowlistModuleAddress,
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'getDefaultAllowlistModuleAddress' is defined but never used.
<!-- __reviewdog__:ChBlODkwZGY0MGJiZWZiNGZlEgZyZGpzb24= -->